league/dashboard/views.py

```diff
@@ -39,24 +39,33 @@ def players():
 
 
 @csrf_protect.exempt
-@blueprint.route('/games/', methods=['GET', 'POST'])
-def games():
+@blueprint.route('/games/', methods=['GET'])
+def list_games():
+    """List all games."""
+    form = GameCreateForm(request.form, csrf_enabled=False)
+    games = Game.query.all()
+    return render_template('dashboard/games.html', games=games,
+                           game_create_form=form)
+
+
+@csrf_protect.exempt
```

Comment:
If it's for testing purposes, you can set up a different `TestConfig` with `WTF_CSRF_ENABLED = False`. Other things that I've put under this `TestConfig` are things like `SECURITY_PASSWORD_HASH = 'plaintext'` to prevent 1-second long bcrypt hash times for each dummy user created; `MAIL_SUPPRESS_SEND = True` to avoid sending email from tests, and so on.